internal/flink/command_artifact_create.go

```diff
@@ -58,10 +71,18 @@ func (c *command) createArtifact(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	runtimeLanguage, err := cmd.Flags().GetString("runtime-language")
+	if err != nil {
+		return err
+	}
+	if _, ok := allowedRuntimeLanguages[runtimeLanguage]; !ok {
+		return fmt.Errorf("invalid value for --runtime-language %s. Allowed values are %v", runtimeLanguage, utils.ArrayToCommaDelimitedString(maps.Keys(allowedRuntimeLanguages), "or"))
+	}
```

Comment:
have we already discussed making validation at server side?

Reply:
Removed it.